waflog/read.go

```diff
@@ -28,7 +28,7 @@ func (ll *FTWLogLines) TriggeredRules() []uint {
 	ll.triggeredRulesInitialized = true
 
 	lines := ll.getMarkedLines()
-	regex := regexp.MustCompile(`\[id "(\d+)"\]|"id":\s*"?(\d+)"?`)
+	regex := regexp.MustCompile(`\[id \\?"(\d+)\\?"\]|"id":\s*"?(\d+)"?`)
```

Comment:
Can you add a comment to describe the three different formats we're trying to detect here? I think the regex is now too complex to understand without context.

Reply:
I proposed a regex description with some format examples. Writing it I realized that we would still not support `[\"id\":\"999999\"]`, however, I’m not aware of any engine or connector that outputs this format. Should we add support for consistency, or wait until a real use case arises?

Reply:
Let's support what we know exists.